src/ui/components/CollectionsStructure/DeleteCollectionsWorkbooksContent/DeleteCollectionsWorkbooksContent.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+import block from 'bem-cn-lite';
+import {I18n} from 'i18n';
+
+import './DeleteCollectionsWorkbooksContent.scss';
+
+const i18n = I18n.keyset('component.collections-structure');
+
+const b = block('dl-collection-delete-collections-workbooks-dialog-content');
+
+export type Props = {
+    collectionTitles: string[];
+    workbookTitles: string[];
+};
+
+export const DeleteCollectionsWorkbooksContent = ({collectionTitles, workbookTitles}: Props) => {
+    return (
+        <div className={b()}>
+            {Boolean(collectionTitles.length) && (
+                <React.Fragment>
+                    <div className={b('header')}>
+                        {i18n('label_delete-collections-for-delete')}:
+                    </div>
+                    <div className={b('list')}>
+                        {collectionTitles.map((title) => (
+                            <div key={title} className={b('title')}>
+                                {title}
```

Comment:
Let's add icons for collections and workbooks

<img width="498" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/77c218ab-1b8d-466d-b5f5-62f302bdf10e">

Like in access dialog:

<img width="437" alt="image" src="https://github.com/datalens-tech/datalens-ui/assets/29424746/2f0f1f5c-896d-4b20-bbbd-169710db61ad">